server/athenian/api/controllers/settings_controller.py

```diff
@@ -566,26 +545,38 @@ async def clean_repository_sets():
             model.acc_id == account,
             model.repository_full_name == full_name,
         ))))
+    clean_deployments_tasks = await _clean_logical_deployments_tasks(
+        [full_name, drop_logical_repo(full_name)], account, pdb,
+    )
+    await gather(*tasks, *clean_deployments_tasks, op="_delete_logical_repository/sql")
```

Comment:
Personally I would make the code simpler by making `_clean_logical_deployments..` function execute all queries.
We would then lose a bit parallelism here but I maybe response time is not important in these 2 endpoints